eva-release/src/main/java/uk/ac/ebi/eva/release/dto/ReleaseStatsPerSpeciesDto.java

```diff
@@ -32,22 +32,32 @@
 
     private Long currentRs;
 
+    private Long remappedCurrentRs;
```

Comment:
I'm not sure what the meaning of remapped RS actually is. 
Now that an RS can be present in multiple assembly maybe its status in a species is less clear. 
An RS id can be `current` in one assembly and `remapped` + `split` in another.

Reply:
If I'm interpreting the [count gathering script](https://github.com/EBIvariation/eva-tasks/blob/master/tasks/eva_2750/gather_clustering_counts_from_mongo.py#L32) correctly, I guess this would be RS that were current/active in a previous assembly and have been remapped?

I agree that some of these numbers are a bit confusing per species, we should probably agree whether the species counts will sum over all assemblies for a taxonomy (in which case RS will be double-counted) or I guess only show counts for the target assembly of that taxonomy.